src/main/java/io/stargate/sgv2/jsonapi/api/model/command/table/IndexDesc.java

```diff
@@ -18,6 +18,9 @@ public interface IndexDesc<DefinitionT extends IndexDefinitionDesc<?>> {
   @JsonProperty(TableDescConstants.IndexDesc.NAME)
   String name();
 
+  @JsonProperty(TableDescConstants.IndexDesc.INDEX_TYPE)
+  String indexType();
```

Comment:
I think it'd be enough to have `@JsonProperty` with no value -- if so, would simply use name "indexType". Same is true for other properties here too.
(`@JsonProperty` itself is needed, that is true, but if method name matches intended name need not include "value").